src/Controller/CRUDController.php

```diff
@@ -77,11 +77,10 @@ class CRUDController extends AbstractController
     /**
      * The template registry of the related Admin class.
      *
-     * @var TemplateRegistryInterface
-     *
      * @psalm-suppress PropertyNotSetInConstructor
+     * @phpstan-ignore-next-line
```

Comment:
Why ? It was working before

Reply:
I will keep this one opened, the others I will resolve, to avoid noise, the reasoning is this: https://github.com/sonata-project/SonataAdminBundle/pull/7767#issuecomment-1071317245

Wdyt?